apps/console/src/features/identity-providers/components/wizards/enterprise-idp-create-wizard.tsx

```diff
@@ -335,6 +335,7 @@ export const EnterpriseIDPCreateWizard: FC<EnterpriseIDPCreateWizardProps> = (
                         message: t("console:develop.features.authenticationProvider.notifications." +
                             "addIDP.error.message")
                     });
+                    setTimeout(() => setAlert(undefined), 4000);
```

Comment:
IMO, we better modify the `Alert` component to use the transition component rather. 
https://react.semantic-ui.com/modules/transition/#types-transition

May be we can address this in a different PR.
@jeradrutnam  WDYT?

Reply:
Let's track this change separately.